src/app/layout/header/SwitchLang.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useRef } from 'react';
+
+import { Dropdown } from '../../../packages';
+import { useOutsideClick } from '../../hooks';
+import { Styled } from './styled';
+
+interface Props {
+  isLangActive: boolean;
+  onToggleLang: () => void;
+}
+
+export const SwitchLang: React.FC<Props> = ({ isLangActive, onToggleLang }) => {
+  const switchLangRef = useRef<HTMLDivElement | null>(null);
+
+  useOutsideClick(switchLangRef, () => {
+    if (isLangActive) {
+      onToggleLang();
+    }
+  });
+
+  return (
+    <Styled.SwitchLangWrapper ref={switchLangRef}>
+      <Styled.SwitchLang isActive={isLangActive} onClick={onToggleLang}>
+        En
```

Comment:
I'd extract language values into a separate enum

Reply:
Yes, i will
Now is only as a mock data
i will use react-i18next library
And my translate words will be strored in public folder